aries_vcx/src/utils/encryption_envelope.rs

```diff
@@ -10,13 +10,16 @@ use uuid::Uuid;
 
 use crate::{errors::error::prelude::*, global::settings, utils::constants};
 
+type AriesMessageBytes = [u8];
```

Comment:
unnecessary abstraction IMO. i think as a developer i'd prefer to just see `&[u8]` as the type, rather than `&AriesMessageBytes`

Reply:
Agreed.

Reply:
I can change that. I added it initially for explicitly documenting what kind of bytes are expected. (Since we wouldn't want someone to pass in byte slices that are not an AriesMessage). 

But since I have also added documentation comments to the function mentioning the same, I suppose this is redundant.